src/components/NoticeOfWithdrawal/ReferenceNumber.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div id="reference-number-section">
+    <v-row no-gutters>
+      <v-col
+        cols="12"
+        sm="3"
+        class="pr-4"
+      >
+        <label
+          id="reference-number-label"
+        >Folio or Reference Number</label>
+      </v-col>
+      <v-col
+        cols="12"
+        :sm="9"
+      >
+        <v-form
+          id="reference-num-form"
+          ref="refNumRef"
+          v-model="valid"
+        >
+          <v-text-field
+            id="reference-number-input"
+            v-model="referenceNumber"
+            label="Folio or Reference Number (Optional)"
+            :rules="referenceNumRules"
+            filled
+          />
+        </v-form>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Emit, Prop, Watch } from 'vue-property-decorator'
+import { FormIF } from '@bcrs-shared-components/interfaces'
+
+  @Component({})
```

Comment:
Remove indent.

Also the styles block below.

Reply:
Done